### PR TITLE
adding support of relative redirect links

### PR DESCRIPTION
**Pull request by kabracity**
Added handling of relative redirected links (url=torrent/... instead of url=http://divxatope/torrent) to reflect recent changes on the page
Minor edits to comply with PEP8

### CODE REVIEW

File: flexget/plugins/urlrewrite_divxatope.py [hunk 1]
```diff
@@ -36,7 +36,15 @@ def parse_download_page(self, url):
             download_link = soup.findAll(href=re.compile('redirect|redirectlink'))
             download_href = download_link[0]['href']
             if "url" in download_href:
-                return download_href[download_href.index('url=') + 4:]
+                redirect_search = re.search('.*url=(.*)', download_href)
+                if redirect_search:
+                        redirect_link = redirect_search.group(1)
```
Comment (gazpachoking) [inline flexget/plugins/urlrewrite_divxatope.py]: Looks like one too many indents here.

File: flexget/plugins/urlrewrite_divxatope.py [hunk 2]
```diff
@@ -36,7 +36,15 @@ def parse_download_page(self, url):
...
+                else:
+                    raise UrlRewritingError('Redirect link for %s could not be found %s' % url)
+                if redirect_link.startswith("http"):
+                    return redirect_link
+                else:
+                    return "http://www.divxatope.com/" + redirect_link
```
Comment (gazpachoking) [inline flexget/plugins/urlrewrite_divxatope.py]: We try to standardize on single quotes `'`, can we switch all the strings to that?

**Comment by kabracity** [comment]
Removed double quotes

**Comment by gazpachoking** [comment]
@kabracity Just that indenting issue still remains. :wink:

**Comment by kabracity** [comment]
My bad, I thought you were referring to an older commit (hadn't noticed miss indentantion at line 41; I hope I'm not missing another one :) )

**Comment by kabracity** [comment]
@gazpachoking any correction else needed from my side?

**Comment by gazpachoking** [comment]
@kabracity Oops, didn't see you fixed it. Thanks!

**Pull request merged**